src/main/java/fi/aalto/cs/apluscourses/model/task/Arguments.java

```diff
@@ -1,25 +1,55 @@
 package fi.aalto.cs.apluscourses.model.task;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
+import org.json.JSONArray;
+
 
 @FunctionalInterface
 public interface Arguments {
-  @Nullable String get(@NotNull String key);
+  @Nullable Object get(@NotNull String key);
 
   /**
    * Gets an argument or throws a runtime exception if the argument is missing.
    * @param key Key
    * @return The (non-null) value of the argument.
    */
   default @NotNull String getOrThrow(@NotNull String key) {
```

Comment:
I suggest renaming methods in this class:
- `getOrThrow` -> `getString`
- `getArrayOrThrow` -> `getArray`
- `get` -> `opt`